src/api/sso/src/slack-bolt-client.js

```diff
@@ -0,0 +1,60 @@
+const { logger } = require('@senecacdot/satellite');
+const { App } = require('@slack/bolt');
+
+const SLACK_BOT_TOKEN = 'slack-bot-token-placeholder';
+const SIGNING_SECRET = 'signing-secret-placeholder';
+const CHANNEL_ID = 'channel-id-placeholder';
```

Comment:
These are probably coming from `process.env`, like we do [here](https://github.com/Seneca-CDOT/telescope/blob/dc899cc3dd06849f9679cc90be1eb05e8c10cb5a/src/api/sso/src/middleware.js#L13). Also, I think it'd be a good idea to add them to `env.local`.